main.c

```diff
@@ -1,25 +1,107 @@
 #include "pico/stdlib.h"
 #include "pico/cyw43_arch.h"
+#include "hardware/pwm.h"
+#include "modes.h"
 
 
 #define GPIO_ON 1
 #define GPIO_OFF 0
+#define PWM_GPIO_PIN 14
+#define BUTTON_GPIO_PIN 15
 
+#define DIVIDER 10
 
 int main(){
 
+    //Initialize wifi module and turn on LED to indicate program is loaded and working
+    cyw43_arch_init();
+    cyw43_arch_gpio_put(CYW43_WL_GPIO_LED_PIN, GPIO_ON);
+
+
+    //Set GPIO 14 pin to PWM mode
+    gpio_set_function(PWM_GPIO_PIN, GPIO_FUNC_PWM);
+
+    //Get PWM channel for that pin;
+    uint slice_number = pwm_gpio_to_slice_num(PWM_GPIO_PIN);
+
+    //Turn PWM on
+    pwm_set_enabled(slice_number, true);
+
+    //Set the PWM wrap point for 2kHz
+    int pwm_wrap_point = 62500;
```

Comment:
lets make #define 2KHZ_WRAP_POINT 62500 at the beginning, it'll be easier to read